alcs-frontend/src/app/services/application/application.dto.ts

```diff
@@ -140,6 +140,25 @@ export interface ApplicationSubmissionDto {
   subdAgricultureSupport?: string;
   subdIsHomeSiteSeverance?: boolean;
   subdProposedLots: ProposedLot[];
+
+  //Soil Fields
+  soilIsNOIFollowUp?: boolean | null;
```

Comment:
I wonder if it worth starting to split interface into smaller ones and use inheritance instead of comment separation?

Reply:
It might be, but since we have 3-4 copies of this DTO I will leave it to its own discussion / ticket.